### DISCUSSION TITLE
How to use multiple query by useQuery from @vue/apollo-composable

### DISCUSSION THREAD

**Discussion by nagaky4** [Akryum/vue-apollo#1276] [Q&A]
Hi everybody
I want to call multiple query at the same time, I try to use useQuery hook twice times but it not working, could you help me please, thanks.
![image](https://user-images.githubusercontent.com/31811450/140082894-dc96eb7e-5f96-4118-82d1-9b42364045a9.png)
![image](https://user-images.githubusercontent.com/31811450/140083193-5d32d0ce-ab50-4c6b-854d-7082063099ac.png)

**Comment by Akryum** [comment]
I don't think your issue is related to having multiple queries.

**Reply by nagaky4** [reply]
hi @Akryum , when i comment one of them, it works fine, otherwise, it throw that error?
 I want to call the first query in onMounted Hook, and the second in a function trigger from button? 
 Are you have any solution, thanks a lot.

**Reply by Akryum** [reply]
It means you have an issue in the second query

**Comment by chulucninh09** [comment]
Can you share your graphql definitions?

**Reply by nagaky4** [reply]
hi @chulucninh09 , i think it is not caused by the graphql schema, here is my schema
![image](https://user-images.githubusercontent.com/31811450/140087829-e4db87f2-ce9b-4038-851e-e6b1c8fc6122.png)
![image](https://user-images.githubusercontent.com/31811450/140087909-2fbc3124-eaa0-45e4-9274-a8c1d3e8e657.png)

**Reply by Akryum** [reply]
You should not declare `input` types client-side, those are part of server Schema definition.
Also you should probably not name your queries `Query` it's pretty confusing.

**Reply by Akryum** [reply]
![image](https://user-images.githubusercontent.com/2798204/140088469-96fcd7dc-0afc-4d58-b446-44af95917c31.png)

**Reply by nagaky4** [reply]
@Akryum  i have updated my query and it works fine thanks a lot. 
one more thing, I want to call the query in mounted hook or in another function, I think we can't do it with the useQuery.

**Reply by Akryum** [reply]
Try `useLazyQuery`

**Reply by nagaky4** [reply]
Thanks a lot @Akryum  🎉🎉🎉
